src/shared/Gitee/GiteeConstants.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+
+namespace Gitee
+{
+    public static class GiteeConstants
+    {
+        public static readonly Uri GiteeDotCom = new Uri("https://gitee.com");
+
+        public const string DefaultAuthenticationHelper = "Gitee.UI";
+
+        // Owned by https://gitee.com/maikebing/git-credential-manager
```

Comment:
Looks great, we'll try to create the same organization for https://github.com/GitCredentialManager on gitee.com.

Reply:
> We might want to see if we can claim `https://gitee.com/GitCredentialManager`. I _think_ this would need to be achieved by creating a "group" (which from what I can tell is the equivalent of a GitHub organization). I'm running into some language issues, as I walk through the process though and don't seem to have a translate option: !

Looks great, we'll try to create the same organization for https://github.com/GitCredentialManager on gitee.com.